@ndlib/gatsby-theme-marble/src/components/App/Pages/Portfolio/PortfolioBody/PortfolioEdit/EditItems/EditItem/index.js

```diff
@@ -1,34 +1,27 @@
+/** @jsx jsx */
+// eslint-disable-next-line no-unused-vars
```

Comment:
I don't think you need this disable?

Reply:
For some reason, when you add in the `/** @jsx jsx */` line which is required for the module, the linter starts to complain about React unless you use a `<React.Fragment />` or other such literal instance of the word `React`.